lib/screens/play/current_card/card_actions/card_actions.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+
+import 'package:treasure_of_the_high_seas/model/card/action/card_action.dart' as Model;
+
+import './card_action.dart';
+
+class CardActions extends StatelessWidget {
+  final List<Model.CardAction> actions;
+
+  CardActions(this.actions);
+
+  @override
+  Widget build(BuildContext context) {
+    
+    return Row(
+      children: [
+        Expanded(
+          flex: 4,
+          child: CardAction(
+            actions.asMap().containsKey(0) ? actions[0] : null
+          ),
+        ),
+        Expanded(
+          flex: 4,
+          child: CardAction(
+            actions.asMap().containsKey(1) ? actions[1] : null
+          ),
+        ),
+        Expanded(
+          flex: 4,
+          child: CardAction(
+            actions.asMap().containsKey(2) ? actions[2] : null
+          ),
+        ),
```

Comment:
Thinking that instead of this we could do a `children.foreach(` block to just add as many `Expanded(...)` jobbers as needed?

Reply:
Oh, but I guess that would lose us having empty panel(s) where there are fewer than 3 actions. But I think that would probably be a better UX to only take up as much space as needed each time?

Reply:
It looked like there was maximum 3 options on every card, and I thought we could just display a blank action if a card had less than 3?

Reply:
Added new issue for this